com.reprezen.swagedit/src/com/reprezen/swagedit/model/Model.java

```diff
@@ -84,6 +97,33 @@ public static Model parseYaml(SwaggerSchema schema, String text) {
         return model;
     }
 
+    public static Iterable<Model> parseYaml(Iterable<IFile> files) {
+        if (files == null || Iterables.isEmpty(files)) {
+            return Lists.newArrayList();
+        }
+
+        final SwaggerSchema schema = Activator.getDefault().getSchema();
+        final ObjectMapper mapper = createMapper();
+
+        final List<Model> modelSet = Lists.newArrayList();
+        for (IFile file : files) {
+            Model model = new Model(schema, file.getFullPath());
+            try {
+                mapper.reader() //
+                        .withAttribute(ATTRIBUTE_MODEL, model) //
+                        .withAttribute(ATTRIBUTE_PARENT, null) //
+                        .withAttribute(ATTRIBUTE_POINTER, JsonPointer.compile("")) //
+                        .withType(AbstractNode.class) //
+                        .readValue(file.getLocationURI().toURL());
```

Comment:
`ObjectReader` we construct here is identical to the one created in`parseYaml(SwaggerSchema schema, String text)`. Can we reuse the code?